src/config/env.js

```diff
@@ -62,7 +64,8 @@
         "urn": "urn:mdr16:dataelement:27:1",
         "placeholder": "e.g. C25.1",
         "unit": "",
-        "valueSetUrl": "http://hl7.org/fhir/ValueSet/condition-code"
+        "valueSetUrl": "http://hl7.org/fhir/sid/icd-10-gm"
```

Comment:
This change is already present in the master. Please merge the master in develop and rebase onto it.